src/Entity/BaseEvent.php

```diff
@@ -238,7 +238,9 @@ public function incrementParticipantsCount(int $increment = 1): void
 
     public function decrementParticipantsCount(int $increment = 1): void
     {
-        $this->participantsCount -= $increment;
+        if (0 < $this->participantsCount) {
```

Comment:
good but not perfect, with `$this->participantsCount = 1` and `$increment = 2` it'll be negatif.